HISTORY.md

```diff
@@ -8,6 +8,8 @@
 * When user uses options.force_consistency_check in RocksDb, instead of crashing the process, we now pass the error back to the users without killing the process.
 * Added experimental ColumnFamilyOptions::sst_partitioner_factory to define determine the partitioning of sst files. This helps compaction to split the files on interesting boundaries (key prefixes) to make propagation of sst files less write amplifying (covering the whole key space).
 * Option `max_background_flushes` can be set dynamically using DB::SetDBOptions().
+* Allowing compaction filter on flushing, picked from [facebook/rocksdb#pr8243](https://github.com/facebook/rocksdb/pull/8243).
+* Remove unused struct `CompactionFilterContext`.
```

Comment:
File it under `Public API change`, as original PR?

Reply:
Solved!